libraries/tlv-account-resolution/src/seeds.rs

```diff
@@ -53,6 +57,7 @@ pub enum Seed {
     ///     * 1 - Discriminator
     ///     * 1 - Start index of instruction data
     ///     * 1 - Length of instruction data starting at index
+    #[cfg_attr(feature = "serde-traits", serde(alias = "instruction_data"))]
```

Comment:
I still highly recommend just sticking with camel-case, because we *will* forget to add the snake-case alias at some point and it will be annoying to debug.